core/src/main/java/com/radixdlt/rev2/REv2StateComputer.java

```diff
@@ -158,7 +158,7 @@ public StateComputerLedger.StateComputerResult prepare(
             proposedTransactions,
             UInt64.fromNonNegativeLong(roundDetails.epoch()),
             UInt64.fromNonNegativeLong(roundDetails.roundNumber()),
-            UInt64.fromNonNegativeLong(roundDetails.proposerTimestampMs()));
+            roundDetails.proposerTimestampMs());
```

Comment:
Argh sorry I resolved this a different way on `betanet` / `main` - continued using a UInt64 on the boundary and casted inside the state manager.

I'm happy for us to resolve it either way around